docs/content/usage/packages/debian.en-us.md

```diff
@@ -77,14 +77,14 @@ curl --user your_username:your_password_or_token \
 ```
 
 If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password.
-You cannot publish a file with the same name twice to a package. You must delete the existing package version first.
+You cannot publish a package if a package of the same name and version already exists. You must delete the existing package first.
```

Comment:
The Debian registry works with files. A package is rejected if it has the same combination of name, version, distribution, component and architecture.